backend/alembic/versions/2023-01-10-07-24-50_e47196be4c5c_add_materialize_view_for_advance_filter.py

```diff
@@ -0,0 +1,41 @@
+"""add_materialize_view_for_advance_filter
+
+Revision ID: e47196be4c5c
+Revises: d37375e79c27
+Create Date: 2023-01-10 07:24:50.778049
+
+"""
+from alembic import op
+from alembic_utils.pg_materialized_view import PGMaterializedView
+
+
+# revision identifiers, used by Alembic.
+revision = "e47196be4c5c"
+down_revision = "d37375e79c27"
+branch_labels = None
+depends_on = None
+
+advance_filter_with_multiple_option_value = PGMaterializedView(
+    schema="public",
+    signature="advance_filter",
+    definition="""
+    SELECT tmp.data,
```

Comment:
The view does not work as we need when filtering with questions from the monitoring form.
I think, here we need to add `identifier` value from `data` table. So we need to alter this view to include `identifier` column.